apps/alpha-test-app/src/views/LayoutDemoView.vue

```diff
@@ -1,453 +1,134 @@
-      'app--detail-footer': settings.footer.showDetailFooter,
-      'app--full-footer': settings.footer.showFullFooter,
-      'app--sidebar': settings.sideBar.showSideBar,
-    }"
-  >
-    <!----------- GRID top row ----------->
-    <div class="demo nav-bar">
-      <strong>Top nav bar</strong> |
-
-      <!-- demo flyout -->
-      <label>
-        Demo Drop Down:
-        <span class="flyout-parent">
-          <input @click="settings.content.showFlyout = !settings.content.showFlyout" />
-          <div v-if="settings.content.showFlyout" class="demo flyout">
-            <LayoutSettings v-model="settings" :show="['content']" />
-          </div>
-        </span>
-      </label>
-
-      |
-      <!-- demo tooltip -->
-      <span class="demo tooltip" :class="{ 'tooltip--forced': settings.content.forceTooltip }">
-        Tooltip parent
-        <div class="tooltip__text">Hello world Hello world Hello world Hello world</div>
-      </span>
-    </div>
-
-    <!----------- GRID sidebar (left col) ----------->
-    <div v-if="settings.sideBar.showSideBar" class="demo side-bar">
-      <LayoutSettings v-model="settings" :show="['content', 'footer', 'sideBar']" />
-
-      <!-- demo tooltip -->
-      <div class="demo tooltip" :class="{ 'tooltip--forced': settings.content.forceTooltip }">
-        Tooltip parent
-        <div class="tooltip__text">Hello world Hello world Hello world Hello world</div>
-      </div>
-    </div>
-
-    <!----------- GRID main (right col) ----------->
-    <div class="page" :class="{ 'page--full-height': !settings.footer.showFullFooter }">
-      <div class="page__content">
-        <OnyxHeadline is="h1">Scrollable page content</OnyxHeadline>
-
-        <LayoutSettings v-model="settings" horizontal />
-        <!-- <LayoutDemoOptions /> -->
-        <!-- demo sticky content -->
-        <div v-if="settings.content.showStickyContent" class="demo sticky-content">
-          Sticky content. Example: Breadcrumb bar, Table header, Headlines
-        </div>
+const isSidebarOpen = ref(true);
 
-        <!-- demo flyout -->
-        <p>
-          <label>
-            Drop Down:
-            <span class="flyout-parent">
-              <input @click="settings.content.showFlyout = !settings.content.showFlyout" />
-              <div v-if="settings.content.showFlyout" class="demo flyout">
-                <LayoutSettings v-model="settings" :show="['content']" />
-              </div>
-            </span>
-          </label>
-        </p>
-
-        <!-- demo tooltip -->
-        <div class="demo tooltip" :class="{ 'tooltip--forced': settings.content.forceTooltip }">
-          Tooltip parent
-          <div class="tooltip__text">Hello world Hello world Hello world Hello world</div>
-        </div>
+const showTempSidebarOpen = computed<boolean>(() => {
+  const { showTempOverlayTransparent, showTempOverlay } = settings.value.sidebar;
+  return (isSidebarOpen.value && (showTempOverlay || showTempOverlayTransparent)) || false;
```

Comment:
```suggestion
  return (isSidebarOpen.value && (showTempOverlay || showTempOverlayTransparent)) ?? false;
```

I think `??` is safer for booleans